test/datasets/commonvoice_test.py

```diff
@@ -0,0 +1,72 @@
+import os
+import csv
+import random
+
+from torchaudio.datasets import commonvoice
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+class TestCommonVoice(TempDirMixin, TorchaudioTestCase):
+    backend = 'default'
+
+    root_dir = None
+    data = []
+    _headers = [u"client_ids", u"path", u"sentence", u"up_votes", u"down_votes", u"age", u"gender", u"accent"]
+    # Note: extension is changed to wav for the sake of test
+    # Note: the first content is missing values for `age`, `gender` and `accent` as in the original data.
+    _train_csv_contents = [
+        ["9d16c5d980247861130e0480e2719f448be73d86a496c36d01a477cbdecd8cfd1399403d7a77bf458d211a70711b2da0845c",
+            "common_voice_en_18885784.wav",
+            "He was accorded a State funeral, and was buried in Drayton and Toowoomba Cemetery.", "2", "0", "", "", ""],
+        ["c82eb9291328620f06025a1f8112b909099e447e485e99236cb87df008650250e79fea5ca772061fb6a370830847b9c44d20",
+            "common_voice_en_556542.wav", "Once more into the breach", "2", "0", "thirties", "male", "us"],
+        ["f74d880c5ad4c5917f314a604d3fc4805159d255796fb9f8defca35333ecc002bdf53dc463503c12674ea840b21b4a507b7c",
+            "common_voice_en_18607573.wav",
+            "Caddy, show Miss Clare and Miss Summerson their rooms.", "2", "0", "twenties", "male", "canada"],
+    ]
+    _folder_audio = "clips"
+    sample_rate = 48000
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = os.path.join(cls.get_base_temp_dir(), 'waves_commonvoice')
```

Comment:
What is this `'waves_commonvoice'`? If it's at root level, it does not matter for Dataset, so it's redundant.